tests/src/Functional/OgMultipleSelectionWidgetAutoCompleteTest.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Drupal\Tests\og\Functional;
+
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Testing the audience field with multiple auto select widgets.
+ *
+ * @group og
+ */
+class OgMultipleSelectionWidgetAutoCompleteTest extends BrowserTestBase {
+
+  use ContentTypeCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['node', 'og'];
+
```

Comment:
3x: Drupal\Tests\BrowserTestBase::$defaultTheme is required in drupal:9.0.0 when using an install profile that does not set a default theme. See https://www.drupal.org/node/3083055, which includes recommendations on which theme to use.
    1x in OgMultipleSelectionWidgetAutoCompleteTest::testAutoCompleteForNonGroupMember from 

```suggestion

  /**
   * {@inheritdoc}
   */
  protected $defaultTheme = 'stark';

```